### ISSUE TITLE
How to use BingRewards with 2FA?

### ISSUE THREAD

**Issue by piersandro** [Prem-ium/BingRewards#86]
I recently enabled 2FA on my account and clearly the bot can no longer perform its automations.
Is there any way I can use it with 2FA enabled?

**Comment by meghrathod** [comment]
I don't think that would be possible when using headless, Non-headless mode, you can quickly put in your 2FA credentials to do it .

**Comment by piersandro** [comment]
Thank you @joeg1307 this is a good workaround. I will try it!

This however has no impact on other accounts if the 2FA is not active, correct?

**Comment by joeg1307** [comment]
@piersandro That's correct. And if you miss the approval of the code it will continue as normal. It will run the searches without being registered to your account, but at least it doesn't stop the whole thing and you can come back an try again. What I've been doing is, on the .env on the `LOGIN` environment variable, at the very beginning of the string I set the account with 2FA so that's the first account to be handled. 

In other words you can work also with the order of accounts set in the `LOGIN` environment variable to better fit to your timing needs of approving the 2FA Codes.
